### PR TITLE
hotfix(Breadcrumb): content with children and example name

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR renames the `BreadcrumbExample` to `BreadcrumbExampleBasic` to avoid the warning that is missing some example and fixes the console warning about `children` and `content` conflicting on `BreadcrumbDivider`

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbDivider.tsx [hunk 1]
```diff
@@ -85,4 +86,9 @@ BreadcrumbDivider.defaultProps = {
   accessibility: breadcrumbDividerBehavior,
 };
 
-BreadcrumbDivider.propTypes = commonPropTypes.createCommon();
+BreadcrumbDivider.propTypes = {
+  ...commonPropTypes.createCommon({
+    content: false,
+  }),
+  content: customPropTypes.shorthandAllowingChildren,
+};
```
Comment (layershifter) [inline packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbDivider.tsx:94]: This change is not valid. `children` and `content` cannot be used in the same time

From which example a warning comes?

Comment (assuncaocharles) [reply packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbDivider.tsx:94]: It comes from usage `<Breadcrumb.Divider><Icon /></Breadcrumb.Divider>` and having `content: '/'` as `defaultProps`

Comment (layershifter) [reply packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbDivider.tsx:94]: In this case please change it to `Breadcrumb.defaultProps.children`

Comment (assuncaocharles) [reply packages/fluentui/react-northstar/src/components/Breadcrumb/BreadcrumbDivider.tsx]: Please, check it now

**Review by layershifter — APPROVED** [review]

**Pull request merged**